mono/eglib/glib.h

```diff
@@ -1,5 +1,17 @@
 #ifndef __GLIB_H
 #define __GLIB_H
+
+// Ask stdint.h for the full C99 features even for C++98 CentOS 6 g++ 4.4.
+#if defined(__cplusplus) && !defined(__STDC_LIMIT_MACROS)
```

Comment:
I think this whole block needs to be wrapped in `#if defined (__cplusplus) && (cplusplus <= 199711L)`
Because in C++11 this is no longer necessary (?) https://stackoverflow.com/a/26523878/570682

Reply:
I’ll have to try centos6 again.

Reply:
It is needed, for the centos6/gcc4.4 C++ prerelease.
Though include cstdint maybe the nicer workaround.
That is where the described compiler-supplied workaround is.